items.md

```diff
@@ -137,6 +137,172 @@ Example: <https://dspace7.4science.it/dspace-spring-rest/#https://dspace7.4scien
 
 It returns the collection where the item belong to
 
+### Mapping Collections
+**GET /api/core/items/<:uuid>/mappingCollections**
```

Comment:
It is an association. This should be included in the list of links returned by the single item endpoint.
It is important to clarify if the returned list include or not the owningCollection. I have no strong preference about it: probably excluding the owningCollection from the returned list could simplify some use cases on the client side but I guess it will add extra effort on the rest implementation or at least some "hack" on the code.

Reply:
I agree with @abollini.

Also, I wonder if this should simply be called `collections` in the `links` on an Item...*especially* if we have it also include the `owningCollection`.  I don't see a reason to actually use the term "mapping" or "mapped" in the REST API *unless* we really need to distinguish it from `owningCollection`.

Reply:
For the "included in the list of links returned by the single item endpoint", are you implying the HAL link to mappingCollections as displayed in
https://dspace7-internal.atmire.com/rest/#/rest/api/core/items/a649a58a-6b5a-4c15-9151-093cd6841491
Would you just want us to also update the item Rest Contract to contain this HAL links in the sample JSON?

The mapping collections don't include the owning collection, we will mention that explicitly in the Rest Contract

The reason to be explicit about this being mapping collections is to also easily allow a UI to add/remove mapping collections for an item. The difference with the owning collection is important here

Reply:
ok